Lib/test/test_modulefinder.py

```diff
@@ -364,7 +380,7 @@ def test_extended_opargs(self):
 b.py
 """ % list(range(2**16))]  # 2**16 constants
         self._do_test(extended_opargs_test)
-
+    
```

Comment:
you'll want ot rebase out this whitespace change as well

Reply:
Ok, will do it!

Reply:
Done